CONTRIBUTING.md

```diff
@@ -0,0 +1,74 @@
+# Contributing Guidelines
+
+The Microsoft acs-engine project accepts contributions via GitHub pull requests. This document outlines the process to help get your contribution accepted.
+
+## Contributor License Agreements
+
+We'd love to accept your patches! Before we can take them, we have to jump a
+couple of legal hurdles.
+
+The [Microsoft CLA](https://cla.microsoft.com/) must be signed by all contributors. Please fill out either the individual or corporate Contributor License Agreement (CLA). Once you are CLA'ed, we'll be able to accept your pull requests. Once you are CLA'ed, we'll be able to accept your pull requests.
+
+***NOTE***: Only original source code from you and other people that have
+signed the CLA can be accepted into the repository.
+
+
+## Support Channels
```

Comment:
Yes. As a Microsoft project some customers have an EE ration of "support". This section should essentially say "This is an open source project and as such no formal support is available. However, like all god open source projects we do offer "best effort" support through github issues."